table/format.h

```diff
@@ -182,24 +182,54 @@ Status ReadFooterFromFile(RandomAccessFileReader* file,
 static const size_t kBlockTrailerSize = 5;
 
 struct BlockContents {
-  Slice data;           // Actual contents of data
-  bool cachable;        // True iff data can be cached
+  Slice data;     // Actual contents of data
+  bool cachable;  // True iff data can be cached
```

Comment:
Typo: cacheable :)

Reply:
This is not my code, an auto-formatter just decided to indent the comments. There are many occurrences of 'cachable' in the codebase, this is not my battle to fight :)